pkg/mountutil/mountutil_windows_test.go

```diff
@@ -78,3 +78,45 @@ func TestParseVolumeOptions(t *testing.T) {
 		})
 	}
 }
+
+func TestProcessSplit(t *testing.T) {
+	tests := []struct {
+		res      Processed
+		src, dst string
+		options  []string
+		s        string
+		wants    []string
+	}{
+		{
+			res:     Processed{},
+			src:     "",
+			dst:     "",
+			s:       "C:\\ProgramData\\nerdctl\\test\\volumes\\default\\testVol\\_data:C:\\src",
+			options: []string{},
+		},
+		{
+			res:     Processed{},
+			src:     "",
+			dst:     "",
+			s:       "C:\\ProgramData\\nerdctl\\test\\volumes\\default\\Volmnt:C:\\src",
```

Comment:
Can we test the split result? like `expected: []string{"C:\\ProgramData\\nerdctl\\test\\volumes\\default\\Volmnt", "C:\\src"}`

Reply:
The test itself is about splitting the string into source and destination for windows.